tests/rule_based_profiler/test_rule_based_profiler.py

```diff
@@ -1132,6 +1163,36 @@ def test_get_profiler(
     assert isinstance(profiler, RuleBasedProfiler)
 
 
+@mock.patch("great_expectations.data_context.data_context.DataContext")
+def test_save_profiler(
+    mock_data_context: mock.MagicMock,
+    populated_profiler_store: ProfilerStore,
+    profiler_config_with_placeholder_args: RuleBasedProfilerConfig,
+):
+    with mock.patch(
+        "great_expectations.data_context.store.profiler_store.ProfilerStore.set",
+        return_value=profiler_config_with_placeholder_args,
+    ):
+        mock_data_context.save_profiler(
+            profiler=profiler_config_with_placeholder_args,
+            profiler_store=populated_profiler_store,
```

Comment:
This is not actually part of the signature right? I think we might want to brainstorm how best to test this functionality.